sycl/test-e2e/AddressSanitizer/out-of-bounds/private/multiple_private.cpp

```diff
@@ -0,0 +1,76 @@
+// REQUIRES: linux, cpu
+// RUN: %{build} %device_asan_flags -Xarch_device "-mllvm=-asan-spir-privates=1" -DVAR=1 -O2 -g -o %t1
+// RUN: env SYCL_PREFER_UR=1 UR_LAYER_ASAN_OPTIONS=detect_privates:1 %{run} not %t1 2>&1 | FileCheck --check-prefixes CHECK,CHECK-VAR1 %s
```

Comment:
I don't think that you need `SYCL_PREFER_UR` env variable. All existing PI plugins should be no-ops at this point, they simply redirect everything to UR.

Reply:
Until PI is fully replaced with UR this is required in order to actually use the UR loader, the sanitizer support is implemented as a UR layer which so is only loaded via the UR loader. It's necessary, for now, to actually test the sanitizer support.

Reply:
> Until PI is fully replaced with UR this is required in order to actually use the UR loader, the sanitizer support is implemented as a UR layer which is only loaded via the UR loader. It's necessary, for now, to actually test the sanitizer support.

Oh, it is more complicated than I though. Got it, thanks for clarification